about.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>cc global submit</title>
+    <link rel="stylesheet" href="./css/about.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="top-navbar">
+        <ul>
+          <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+          <li><a href="#"><i class="fa fa-facebook"></i></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My page</a></li>
+          <li><a href="#"> Logout</a></li>
+        </ul>
+      </div>
+
+      <div class="main-navbar">
+        <div class="logo">
+          <a href="#"><img src="./img/consul-logo.png" alt="" /></a>
+        </div>
```

Comment:
- [ ] Please, check through your home and about pages, to ensure that all `alt` attributes of the `<img>` tags are given befitting values. Leaving it blank is a bad practice, as it limits the accessibility of the page.